README.md

```diff
@@ -1,79 +1,87 @@
-## Security
+# Preview: AWS Amplify's new code-first DX (Gen 2) for building backends
 
-See [CONTRIBUTING](CONTRIBUTING.md#security-issue-notifications) for more information.
+This next generation of Amplify’s backend building experience lets you author your frontend and backend definition completely with TypeScript, a file convention, and Git branch-based environments, is now available in Preview. To learn more, please visit [AWS Amplify (Gen 2)](https://docs.amplify.aws/gen2/).
```

Comment:
Would customers know what we mean by "file convention" here?

Reply:
Fixed linting. The file convention is the same blurb from the docs